main.py

```diff
@@ -183,7 +190,25 @@ def run():
 
     log("Server started!")
 
+def startMonitorsIfNeeded():
+    dir = os.path.dirname(__file__)
+    prog = os.path.join(dir, 'main.py')
+    scheduler = CronScheduler()
+
+    # Temp if on RasPi
+    if c.Temp.exists():
+        log('Start monitor of CPU temp...')
+        critical_events_command = 'python3 {} --critical-events'.format(prog)
+        scheduler.createRecurringJob(
+            '* * * * *', 
```

Comment:
Also, I'm also wondering if we should offer the ability for the user to clarify how frequently they want use to check their CPU temp.